middlewares/authenticate.js

```diff
@@ -0,0 +1,30 @@
+const jwt = require("jsonwebtoken");
+const { User } = require("../models/users");
+const { HttpError } = require("../utils");
+
+const { SECRET_KEY } = process.env;
+
+const authenticate = async (req, res, next) => {
+  const { authorization = "" } = req.headers;
+  const [bearer, token] = authorization.split(" ");
+
+  if (bearer !== "Bearer") {
+    next(HttpError(401, "Not authorized"));
```

Comment:
```suggestion
    return next(HttpError(401, "Not authorized"));
``` тут в аналогічних випадках